packages/macro/tests/__snapshots__/index.test.ts.snap

```diff
@@ -81,16 +92,20 @@ exports[`macros 3. custom-wepback-chunk-client-side: 3. custom-wepback-chunk-cli
 
 import { useSSRComputation } from "../lib/index.macro"
 const x = useSSRComputation("./a.ssr-computation", [], { webpackChunkName: "custom-chunk-name" })
-  
+    
 
       ↓ ↓ ↓ ↓ ↓ ↓
 
+import { useCallback } from "react";
 import useSSRComputation_Client from "use-ssr-computation.runtime/lib/useSSRComputation_Client";
 const x = useSSRComputation_Client(
-  () =>
-    import(
-      /* webpackChunkName: "custom-chunk-name-ssr-computations" */ "./a.ssr-computation"
-    ),
+  useCallback(
+    () =>
+      import(
+        /* webpackChunkName: "custom-chunk-name-ssr-computations" */ "./a.ssr-computation"
+      ),
+    [],
+  ),
```

Comment:
Can we define this function at the module scope and used it without `useCallback`?
E.G:
```
import { useCallback } from "react";
import useSSRComputation_Client from "use-ssr-computation.runtime/lib/useSSRComputation_Client";
function importCustomChunkNameSsrComputation() {
  return import(/* webpackChunkName: "custom-chunk-name-ssr-computations" */ "./a.ssr-computation");
}

// ...

const x = useSSRComputation_Client(importCustomChunkNameSsrComputation,[]),
```

Reply:
@Romex91 Done